ambry-store/src/test/java/com.github.ambry.store/BlobStoreCompactorTest.java

```diff
@@ -71,7 +71,7 @@
 
   private MetricRegistry metricRegistry;
```

Comment:
no tests for the direct IO mode?

Reply:
Some tests needed for directIO mode @zzmao

Reply:
Tests added.